integration_test/functions/src/index.ts

```diff
@@ -33,8 +34,37 @@ export const integrationTests: any = functions.https.onRequest((req: Request, re
       admin.auth().deleteUser(userRecord.uid);
     }),
   ]).then(() => {
-    // On test completion, redirect the user to the Firebase RTDB dashboard, to
-    // see the results stored there.
-    resp.redirect(`https://${process.env.GCLOUD_PROJECT}.firebaseio.com/testRuns/${testId}`);
+    // On test completion, check that all tests pass and reply "PASS", or provide further details.
```

Comment:
Eventually we may want to break this into client + server. The HTTP function could return the test ID and then a client would poll for the ID. This would help us avoid stricter cutoffs in maximum HTTP request duration.